client/signup/accordion-form/form-components.tsx

```diff
@@ -246,6 +246,6 @@ export const HorizontalGrid = styled.div`
 	margin-bottom: 20px;
 	flex-wrap: wrap;
 	@media ( min-width: 1100px ) {
-		flex-wrap: nowrap;
+		flex-wrap: wrap;
```

Comment:
Any reason for this change? The `flex-wrap: nowrap;` could be there since we used this component before in another place. If `flex-wrap: nowrap;` is not required anymore, we can simply remove this media query since the default value for `flex-wrap` is `wrap` (L247)

Reply:
Ah yes nice catch let me check this

Reply:
I have checked the other places as well. They are used in the same way. So this part can be removed 👌